### ISSUE TITLE
Hipchat notification setting bug related to utf8 color choice

### ISSUE THREAD

**Issue by paihu** [ansible/awx#585]
##### ISSUE TYPE

 - Bug Report

##### COMPONENT NAME

 - UI

##### SUMMARY

Hipchat Notification bug in Japanese language.

##### ENVIRONMENT
* AWX version: 1.0.1.145
* AWX install method: docker on linux
* Ansible version:  2.4.1.0
* Operating System:
* Web Browser: Google Chrome 56

##### STEPS TO REPRODUCE

1. set language to Japanese.
2. go to configuration hipchat notificaiton.
3. color choice "赤","緑","黄","紫","灰色" or "ランダム".

complete configuration. then param "color" set "赤","緑","黄色","紫","灰色" or"ランダム".
but hipchat notification API's valid value is "red", "green","yellow","purple","gray" or "random".

##### EXPECTED RESULTS

param color set "red","green","yellow","purple","gray" or "random".

##### ACTUAL RESULTS

param color set "赤","緑","黄色","紫","灰色" or"ランダム".

##### ADDITIONAL INFORMATION

```
awx=#select notification_configuration from main_notificationtemplate;

                                              notification_configuration


------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
 {"rooms": ["#test"], "token": "$encrypted$AESCBC$Z0FBQUFBQmFBTmlFSW05N0JkZGpyZTF5bVpUN2FvdzFYUXZsMHdVVzVHSFdOSmQtYWNMSE8yWXRyZ3NmTnBUclJxZWRjQ1VRcnptNlZ4cDE5NFg0bzFjcEl2VThDZ0dRNnc9PQ==", "api_url": "https://hipchat.com", "message_from": "tst", "color": "\u7d2b", "notify": false, "use_ssl": false, "use_tls": false}
(1 row)
```
choice "紫" then  database set "color": "\u7d2b".

**Comment by ahuffman** [comment]
I'm seeing this same issue on Tower 3.2.5 with Hipchat notifications in English version as well.  Found a solution from this issue posted in the Ansible core repo: 
[ansible/ansible issue 41389](https://github.com/ansible/ansible/issues/41389)

The file "/var/lib/awx/public/static/app.js" needs to have all the colors changed from first letter capitalization to all lower case.

**Comment by ryanpetrello** [comment]
@ahuffman a fix for your issue landed in awx awhile back: https://github.com/ansible/awx/pull/301

...and will make its way into the next major version of Ansible Tower.

The issue described here is slightly different and only applies to non-English installs; it will also be fixed in an upcoming version of awx.

**Comment by ryanpetrello** [comment]
This should be fixed in AWX 1.0.7.
